nomad/api/base.py

```diff
@@ -10,12 +10,13 @@ class Requester(object):
 
     ENDPOINT = ""
 
-    def __init__(self, address=None, uri='http://127.0.0.1', port=4646, namespace=None, token=None, timeout=5, version='v1', verify=False, cert=(), region=None, **kwargs):
+    def __init__(self, address=None, uri='http://127.0.0.1', port=4646, namespace=None, token=None, timeout=5, blocking_timeout=360, version='v1', verify=False, cert=(), region=None, **kwargs):
```

Comment:
Would we want to add this to `nomad/__init__.py` to pass this when creating all the endpoint objects so the blocking_timeout could be adjusted on the constructor?

Reply:
Yes we would! That was purely an oversight on my behalf after setting a default which was usable for my purposes.